staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_pause.go

```diff
@@ -93,6 +94,7 @@ func NewCmdRolloutPause(f cmdutil.Factory, streams genericclioptions.IOStreams)
 	o.PrintFlags.AddFlags(cmd)
 
 	usage := "identifying the resource to get from a server."
+	cmd.Flags().StringVarP(&o.LabelSelector, "selector", "l", o.LabelSelector, "Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)")
```

Comment:
Can we please create a `cmdutil.AddLabelSelectorFlagVar` and use it like the ones below this?

A good follow up PR will be to refactor other commands to use it.

Reply:
Will do, I think I can do that in this PR as well.